mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -1222,19 +1229,306 @@ def test_pin_profile(self):
             api_url('/searches/'),
             status=201,
         )
+        responses.add(
+            responses.POST,
+            api_url('/prisoners/1/monitor'),
+            status=204,
+        )
 
         self.login(follow=False)
         self.client.get(
             reverse('security:prisoner_detail', kwargs={'prisoner_id': 1}) +
             '?pin=1'
         )
-        self.assertEqual(
-            json.loads(responses.calls[-1].request.body.decode()),
-            {
-                'description': 'A1409AE JAMES HALLS',
-                'endpoint': '/prisoners/1/credits/',
-                'last_result_count': 4,
-                'site_url': '/en-gb/security/prisoners/1/?ordering=-received_at',
-                'filters': [{'field': 'ordering', 'value': '-received_at'}],
-            },
-        )
+
+        for call in responses.calls:
+            if call.request.path_url == '/searches/':
+                self.assertEqual(
+                    json.loads(call.request.body.decode()),
+                    {
+                        'description': 'A1409AE JAMES HALLS',
+                        'endpoint': '/prisoners/1/credits/',
+                        'last_result_count': 4,
+                        'site_url': '/en-gb/security/prisoners/1/',
+                        'filters': [{'field': 'ordering', 'value': '-received_at'}],
+                    },
+                )
+
+
+class NotificationsTestCase(SecurityBaseTestCase):
+    def test_cannot_access_without_pilot_flag(self):
+        with responses.RequestsMock() as rsps:
+            self.login(follow=False, rsps=rsps)
+            response = self.client.get(reverse('security:notification_list'), follow=True)
+        self.assertNotContains(response, '<!-- security:notification_list -->')
+
+    def login_with_pilot_user(self, rsps):
+        self.login(user_data=self.get_user_data(flags=[
+            notifications_pilot_flag, hmpps_employee_flag, confirmed_prisons_flag,
+        ]), rsps=rsps)
+
+    def test_no_notifications_not_monitoring(self):
```

Comment:
A bit of docstrings for these tests would help as it takes time to figure out that they are expecting.